outputs/costs_2020.csv

```diff
@@ -4,487 +4,519 @@ Ammonia cracker,ammonia-input,1.46,MWh_NH3/MWh_H2,"ENGIE et al (2020): Ammonia t
 Calculated. For a large (2500 t_NH3/d input) facility. Base cost for facility: 51 MEUR at capacity 20 000m^3_NH3/h = 339 t_NH3/d input. Cost scaling exponent 0.67. Ammonia density 0.7069 kg/m^3. Conversion efficiency of cracker: 0.685. Ammonia LHV: 5.167 MWh/t_NH3."
 Ammonia cracker,lifetime,25.0,years,"Ishimoto et al. (2020): 10.1016/j.ijhydene.2020.09.017 , table 7.",
-BioSNG,C in fuel,0.324,per unit,"Stoichiometric calculation, doi:10.1016/j.apenergy.2022.120016",
-BioSNG,C stored,0.676,per unit,"Stoichiometric calculation, doi:10.1016/j.apenergy.2022.120016",
-BioSNG,CO2 stored,0.2479,tCO2/MWh_th,"Stoichiometric calculation, doi:10.1016/j.apenergy.2022.120016",
-BioSNG,FOM,1.608,%/year,"Danish Energy Agency, data_sheets_for_renewable_fuels.xlsx","84 Gasif. CFB, Bio-SNG:  Fixed O&M"
+Battery electric (passenger cars),Efficiency (carrier to wheel),68.0,%,PATHS TO A CLIMATE-NEUTRAL ENERGY SYSTEM The German energy transformation in its social context. https://www.ise.fraunhofer.de/en/publications/studies/paths-to-a-climate-neutral-energy-system.html,Battery electric (passenger cars)
+Battery electric (passenger cars),FOM,0.9,%,PATHS TO A CLIMATE-NEUTRAL ENERGY SYSTEM The German energy transformation in its social context. https://www.ise.fraunhofer.de/en/publications/studies/paths-to-a-climate-neutral-energy-system.html,Battery electric (passenger cars)
+Battery electric (passenger cars),investment,33000.0,EUR/PKW,PATHS TO A CLIMATE-NEUTRAL ENERGY SYSTEM The German energy transformation in its social context. https://www.ise.fraunhofer.de/en/publications/studies/paths-to-a-climate-neutral-energy-system.html,Battery electric (passenger cars)
+Battery electric (passenger cars),lifetime,15.0,years,PATHS TO A CLIMATE-NEUTRAL ENERGY SYSTEM The German energy transformation in its social context. https://www.ise.fraunhofer.de/en/publications/studies/paths-to-a-climate-neutral-energy-system.html,Battery electric (passenger cars)
+Battery electric (trucks),FOM,14.0,%,PATHS TO A CLIMATE-NEUTRAL ENERGY SYSTEM The German energy transformation in its social context. https://www.ise.fraunhofer.de/en/publications/studies/paths-to-a-climate-neutral-energy-system.html,Battery electric (trucks)
+Battery electric (trucks),investment,204067.0,EUR/LKW,PATHS TO A CLIMATE-NEUTRAL ENERGY SYSTEM The German energy transformation in its social context. https://www.ise.fraunhofer.de/en/publications/studies/paths-to-a-climate-neutral-energy-system.html,Battery electric (trucks)
+Battery electric (trucks),lifetime,15.0,years,PATHS TO A CLIMATE-NEUTRAL ENERGY SYSTEM The German energy transformation in its social context. https://www.ise.fraunhofer.de/en/publications/studies/paths-to-a-climate-neutral-energy-system.html,Battery electric (trucks)
+BioSNG,C in fuel,0.32,per unit,"Stoichiometric calculation, doi:10.1016/j.apenergy.2022.120016",
```

Comment:
If you look at the digits for the values, e.g. here `0.32` compared to what it previously was `0.324`. The values are rounded in the script to a certain value and I assume you changed the rounding to 2 digits instead of 4

Reply:
I recreated the output files using this PR and got the 4 digits, so I'm not sure why this is happening. Maybe you open/save the files in excel @BertoGBG ?

Reply:
Hej @lisazeyen 
I realised that I didn't upload the new output files before making a pull request. Anyway I am not sure joy the rounding was changed, I never meant to.
 
not the output files are correct with the vehicles added and when I look at them in my GitHub they have 4 digits rounding. 
I suggest you test the code locally if it does not show 4 digits.

/A